Makefile

```diff
@@ -2,11 +2,18 @@ build:
 	@yarn turbo run build
 build-sequential:
 	@yarn turbo run --concurrency 1 build
+node-deps:
+	@yarn
+	build
```

Comment:
Is this a typo? How does this pass CI? Locally I just get `make[1]: build: No such file or directory`
```suggestion
	@yarn build
```

Reply:
Lol, oops, let me fix that!